### DISCUSSION TITLE
AggregatorV3Interface

### DISCUSSION THREAD

**Discussion by mihirWeb** [Cyfrin/foundry-full-course-cu#1293] [Q&A] [answered]
I am unable to find AggregatorV3Interface.sol file anywhere in the chainlink GitHub docs......here is the link where I've been searching

https://github.com/smartcontractkit/chainlink/tree/develop/contracts/src/v0.8/interfaces

**Comment by johnsonstephan** [comment, marked as answer]
**TL;DR - [AggregatorV3Interface.sol](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol)**

---

Hi @mihirWeb, you can find the `AggregatorV3Interface.sol` file at the following location in the Chainlink GitHub repository: [AggregatorV3Interface.sol](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol). It's a little tricky to locate as it is in the shared folder. 

Does this help to answer your question?

**Reply by mihirWeb** [reply]
Thanks a lot @johnsonstephan

**Reply by johnsonstephan** [reply]
@mihirWeb Happy to help!
